etc/vultisig/fee.yml

```
@@ -1,9 +1,16 @@
+eth_provider: ${ETH_PROVIDER_URL}
+chain_id: 1
+jobs:
+    load:
+        cronexpr: "@every 2m"
+        max_concurrent_jobs: 10
+    transact:
+        cronexpr: "0 12 * * 5"
+        max_concurrent_jobs: 10
+    post:
+        cronexpr: "@every 5m"
+        max_concurrent_jobs: 10
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)